### PR TITLE
Issue 256

### PR DESCRIPTION
**Pull request by astrofle**
fix: add support for AzEl observations

Adds check for empty `RADESYS` column and functionality to handle AltAz (AzEl) coordinate frame

Still needs tests, but it can handle Issue #256.

### CODE REVIEW

**Review by etsmit — CHANGES_REQUESTED** [review]
Two things:
1. Add more to the warning (either at the GBTFITSLoad step or when changing velocity frames) that the Az/El values are invalid and will result in an incorrect frequency shift. This could be detected as a separate case from drift scans by detecting when e.g. elevation <4.9. (Or when az/el values are unchanging?)
2. Possibly add the scan numbers corresponding to the blank radesys rows to the warning message, since the antenna manager can be taken in/out over the course of a session. For example, "No RADESYS specified for scans X,Y,Z; assuming AltAz"

**Comment by astrofle** [comment]
The last update also adds support for HaDEC observations.

File: src/dysh/fits/gbtfitsload.py [hunk 1]
```diff
@@ -1633,3 +1635,28 @@ def write_scans(self, fileobj, scans, output_verify="exception", overwrite=False
...
+
+    def _update_radesys(self):
+        """
+        Updates the 'RADESYS' column of the index for cases when it is empty.
+        """
+
+        radesys = {"AzEl": "AltAz", "HADec": "hadec"}
+
+        warning_msg = (
+            lambda scans, a, coord, limit: f"""Scan(s) {scans} have {a} {coord} below {limit}. The GBT does not go that low. Any operations that rely on the sky coordinates are likely to be innacurate (e.g., switching velocity frames)."""
```
Comment (etsmit) [inline src/dysh/fits/gbtfitsload.py:1647]: "inaccurate"

**Review by etsmit — APPROVED** [review]

**Comment by astrofle** [comment]
@mpound is this fix acceptable?

**Review by mpound — APPROVED** [review]
Looks good to me.

**Pull request merged**